lib/peep.rb

```diff
@@ -0,0 +1,34 @@
+require 'pg'
+
+class Peep
+  attr_reader :id, :content
+
+  def initialize(id:, content:)
+    @id = id
+    @content = content
+  end
+
+  def self.all
+    if ENV['RACK_ENV'] == 'test'
```

Comment:
Can this be refactored as  lines 12 to 16 will/are being used at the start of every method in this class